src/DatePicker/DatePicker.js

```diff
@@ -180,14 +180,14 @@ const DatePicker = React.createClass({
 
   getInitialState() {
     return {
-      date: this._isControlled() ? this._getControlledDate() : this.props.defaultDate,
+      date: this.isControlled() ? this._getControlledDate() : this.props.defaultDate,
```

Comment:
I'm not sure it was even started! (That was the only one removed... stray delete?) I've cleaned up the rest in any case. :+1: